composer.json

```diff
@@ -13,7 +13,8 @@
         "irc": "irc://irc.freenode.org/#librenms"
     },
     "config": {
-        "platform": {"php": "5.3.9"}
+        "platform": {"php": "5.3.9"},
```

Comment:
FYI on this: https://github.com/symfony/symfony-standard/issues/858

I'm not sure we should remove this line here or if not having the lock file is fine. However we should actually ship the composer.lock file: https://ilikekillnerds.com/2014/09/should-you-commit-composer-lock-into-your-git-repository/

Reply:
Yes, we will definitely ship composer.lock.  But it will be in the switch over PR.

The reason we hard-coded platform was to keep our dependencies at older versions.  Now that 5.6 is the minimum, we can remove that as well.  but not yet.

Reply:
Actually, I can put composer.lock here and change that.